sections/main-product.liquid

```diff
@@ -711,6 +732,11 @@
           "label": "t:sections.main-product.blocks.popup.settings.page.label"
         }
       ]
+    },
+    {
+      "type": "rating",
+      "name": "t:sections.main-product.blocks.rating.name",
```

Comment:
@sofiamatulis We'll need to add the info text under the Product rating block inside the product template. There might be no setting but this information would benefit merchants here too!

Reply:
Oh, in this case I believe `paragraph` would do the magic, no?

Reply:
@melissaperreault Thank you! Updated 🙏